packages/lesswrong/components/tagging/FilterMode.tsx

```diff
@@ -82,6 +82,16 @@ const styles = (theme: ThemeType): JssStyles => ({
   }
 });
 
+const handleCustomInput = (input: string) => {
+  const parsed = parseFloat(input);
+  if (Number.isNaN(parsed)) {
+    return 0;
+  }
```

Comment:
I mostly made this comment because of the below comments about 0 and .5

Reply:
I've just pushed quite a large change to this logic where we now pause for 1.5 seconds after entering any standard value before clearing the input. Opinions are very welcome - I think I like the UX, but I could probably be convinced otherwise. This is similar to how vim works, but maybe vim isn't the best place to copy UX from.